tests/integration/examples/test_resource_values.py

```diff
@@ -0,0 +1,55 @@
+"""Examples for SDK Documentation that can be run to test they are working"""
+
+from tests.common import BaseCase
+
+
+@BaseCase._skip_in_ci
+class TestExamples(BaseCase):
+
+    def test_get_and_set_resource_value(self):
+        # an example: get and set a resource value
+        from mbed_cloud.foundation import Device
+        from mbed_cloud import ConnectAPI
+
+        # Use the Foundation interface to find a device.
+        device = Device().list(max_results=1).next()
```

Comment:
it needs to be a connected device, so we need to set a filter device_state__eq=REGISTERED

Reply:
Done